recipe/meta.yaml

```diff
@@ -10,40 +10,49 @@ package:
 source:
   fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
-  sha256: 8f157e7397158812cabd340b68546f1baa55a486ed0aad8bc26877593dc2983d
+  sha256: 3296c9d49b7dcb894b20db5d7d1c1a443912b4d82358e03f836575e8398e0d60
 
 build:
-  skip: true  # [win]
-  number: 202
+  skip: true  # [win32 or (win and py<35)]
+  number: 200
   features:
     - blas_{{ variant }}
+    - vc14    # [win and py>=35]
```

Comment:
This isn't needed as we depend on `python` as a `build` and `run` requirement, which already tracks this for us.